CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+## v0.7.0 - 2021-03-15
+
+We're happy to announce the release of Lokomotive v0.7.0 (Ghan).
+
+### Changes in v0.7.0
+
+#### Kubernetes updates
+
+- Update Kubernetes to v1.20.4 ([#1410](https://github.com/kinvolk/lokomotive/pull/1410)).
+
+#### New components
+
+- Add component `node-problem-detector` ([#1384](https://github.com/kinvolk/lokomotive/pull/1384)).
+
+#### Component updates
+
+- Update `aws-ebs-csi-driver` from `0.7.0` to `0.9.0` ([#1393](https://github.com/kinvolk/lokomotive/pull/1393)).
+
+- Update `web-ui` from `0.1.3` to `0.2.1` ([#1412](https://github.com/kinvolk/lokomotive/pull/1412)).
+
+#### Features
+
+- AWS EBS CSI Driver: Add `node_affinity` and `tolerations` ([#1393](https://github.com/kinvolk/lokomotive/pull/1393)).
+
+- EM: Add worker pool specific `facility` attribute ([#1359](https://github.com/kinvolk/lokomotive/pull/1359)).
+
+#### Documentation
+
+- Use FLUO to update nodes ([#1295](https://github.com/kinvolk/lokomotive/pull/1295)).
+
+- How to add a worker pool in a different facility ([#1361](https://github.com/kinvolk/lokomotive/pull/1361)).
+
+- Refactor AWS quickstart guide ([#1273](https://github.com/kinvolk/lokomotive/pull/1273)).
+
+#### Bug fixes
+
+- EM: Add `Restart=on-failure` and `RestartSec=5s` for the metadata service ([#1362](https://github.com/kinvolk/lokomotive/pull/1362)).
+
+- Fix wrong etcd settings, clean up leftovers from etcd move from rkt to docker based daemon ([#1382](https://github.com/kinvolk/lokomotive/pull/1382)).
+
+- contour: Fix hostPort regression ([#1342](https://github.com/kinvolk/lokomotive/pull/1342)).
+
+#### Deprecation removal
+
+- baremetal: Remove `enable_tls_bootstrap` attribute ([#1380](https://github.com/kinvolk/lokomotive/pull/1380)).
+
+#### Miscellaneous
+
+- Remove a deprecated cert-manager namespace label `certmanager.k8s.io/disable-validation=true` ([#1372](https://github.com/kinvolk/lokomotive/pull/1372)).
```

Comment:
Shouldnt this go in Deprecation removal section?

Reply:
Fixed.